model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -62,9 +62,11 @@ class OrtConfigurationTest : WordSpec({
 
             with(ortConfig.scanner) {
                 archive shouldNotBeNull {
-                    fileStorage.httpFileStorage should beNull()
-                    fileStorage.localFileStorage shouldNotBeNull {
-                        directory shouldBe File("~/.ort/scanner/archive")
+                    fileStorage shouldNotBeNull {
+                        httpFileStorage should beNull()
+                        localFileStorage shouldNotBeNull {
+                            directory shouldBe File("~/.ort/scanner/archive")
+                        }
```

Comment:
This commit should actually add an example for the postgres config to reference.conf and verify here that it can be parsed, but then the `require` above would fail. Maybe we should just print a warning and use the file storage in this case instead of throwing an exception so that reference.conf can serve it's purpose to show all possible configuration options?